### PR TITLE
extend estimation of area under curve of y=x using monte carlo simulation to any given lower and upper bound

### PR DESCRIPTION
**Pull request by singlav**
extend estimation of area under curve of y=x using monte carlo simulation to any given lower and upper bound.

### **Checklist:**
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [ ] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [ ] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Comment by singlav** [comment]
@cclauss @onlinejudge95 please review. Thanks.

File: maths/monte_carlo.py [hunk 1]
```diff
@@ -42,29 +42,34 @@ def area_under_line_estimator(iterations: int,
...
     1. Calls "area_under_line_estimator" function
     2. Compares with the expected value
     3. Prints estimated, expected and error value
     """
-    estimate = area_under_line_estimator(iterations)
+    
+    estimated_value = area_under_line_estimator(iterations, min_value, max_value)
+    expected_value = (max_value*max_value - min_value*min_value) / 2
+    
     print("******************")
```
Comment (onlinejudge95) [inline maths/monte_carlo.py]: As mentioned in the previous PR, instead of printing the results it makes more sense to **return** them

Comment (cclauss) [reply maths/monte_carlo.py]: I think it is OK for the _check() function to print because it is not the algorithm.

**Comment by singlav** [comment]
Thank you so much. @cclauss , @onlinejudge95

**Pull request merged**